app/models/employment/BenefitsViewModel.scala

```diff
@@ -244,8 +237,9 @@ object BenefitsViewModel {
 }
 
 case class EncryptedBenefitsViewModel(
+                        accommodationRelocationModel: Option[EncryptedAccommodationRelocationModel] = None,
```

Comment:
same here - can you move carVanFuelModel so it's before accommodationRelocationModel